scripts/runE2ETests.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+echo "Starting E2E tests..."
+
+echo "Starting Nightwatch tests..."
+npm run --silent e2e:old
+NIGHTWATCH_TESTS_EXIT_CODE=$?
+echo "Finished Nightwatch tests."
+
+echo "Starting Cypress tests..."
+npm run --silent e2e:new
+CYPRESS_TESTS_EXIT_CODE=$?
+echo "Finished Cypress tests."
+
+if [[ $NIGHTWATCH_TESTS_EXIT_CODE -eq 0 ]] && [[ CYPRESS_TESTS_EXIT_CODE -eq 0 ]];
+then
+  echo "All E2E tests passed successfully."
+  exit 0
+else
+  echo "Some E2E tests failed."
+  exit 1
+fi
```

Comment:
```suggestion
#!/usr/bin/env bash
set -xo pipefail

npm run --silent e2e:old
npm run --silent e2e:new
```
The `-x` will print the executed commands, which makes the echos not necessary.
The `-o pipefail` eliminates the need for the logic that starts at line 15.